shotgun_api3/shotgun.py

```diff
@@ -3443,8 +3443,8 @@ def _call_rpc(self, method, params, include_auth_params=True, first=False):
                 # We've seen some rare instances of SG returning 502 for issues that
                 # appear to be caused by something internal to SG. We're going to
                 # allow for limited retries for those specifically.
-                if attempt != max_attempts and e.errcode == 502:
-                    LOG.debug("Got a 502 response. Waiting and retrying...")
+                if attempt != max_attempts and e.errcode in [502, 504]:
```

Comment:
You might want to use HTTPStatus constants instead: `http.HTTPStatus.SERVICE_UNAVAILABLE` and `http.HTTPStatus.GATEWAY_TIMEOUT`

Not a blocker but it's usually better to use constants.